src/20_Components/amp-bind.html

```diff
@@ -0,0 +1,65 @@
+<!---{
+    "experiments": ["amp-bind"],
+    "preview": "default"
+}--->
+<!--
+  #### Introduction
+
+  Allows elements to mutate in response to user actions via data binding and simple JS-like expressions
```

Comment:
* Can you link the offical docs: s/Allows/The [amp-bind](...) component allows/
* s/expressions/expressions./
* Can you maybe list a few use cases what this is good for?